### DISCUSSION TITLE
CO2 dissolution in water

### DISCUSSION THREAD

**Discussion by Santhosh91000** [GeoChemFoam/GeoChemFoam#24] [Q&A] [answered]
Hello all,

Hope everything is well. I am reaching out with a specific inquiry regarding my current project. I am currently engaged in modeling the behavior of CO2 emulsion transported by water within a rectangular T-junction channel. My intention is to employ the interReactiveTransferFoam solver to capture the miscibility dynamics between CO2 and water, which appears to be a suitable choice based on the insights provided in the "2D Cavity" tutorial.

However, I am faced with a particular challenge. At this stage, I do not wish to account for chemical reactions, as the equilibrium constants are remarkably low. Nevertheless, I do intend to consider diffusion and advection effects in my simulation. Consequently, I find myself uncertain about what to specify as "phase 2" within the "thermoPhysicalProperties" file.

Should I designate "inertMultiComponentMixture" with water as phase 2? This seems somewhat counterintuitive to me, given that water serves as the solvent rather than a dilute species. Alternatively, should I simply suppress phase 2 

I kindly request your guidance on how to structure this file effectively to ensure that the solver accurately comprehends my intentions.

I appreciate your time and assistance immensely.
Sincerely
Santhosh

**Comment by julienmaes** [comment, marked as answer]
If there is no reaction, you can use the interTransferFoam solver.

Otherwise use inertMultiComponentMixture - inert here means no reaction

**Comment by Santhosh91000** [comment]
Hello Julian,

Thanks for the fast reply! I am going to check out interTransferFoam
Just to be sure for the second option in interRactiveTransferFoam:
This is my file:
''
Phase1 inertMultiComponentMixture;
Phase2 inertMultiComponentMixture;
PhaseTransfer true;

solutionSpecies
{
        CO2	
	{
		D1 D1 [ 0 2 -1 0 0 0 0 ] 1.6e-7;
		D2 D2 [ 0 2 -1 0 0 0 0 ] 0;
		H H [ 0 0 0 0 0 0 0 ] 1.25;
                Mw Mw [ 1 0 0 0 -1 0 0 ] 44;
	}
        H2O	
	{
		D1 D1 [ 0 2 -1 0 0 0 0 ] 0;
		D2 D2 [ 0 2 -1 0 0 0 0 ] 0;
		H H [ 0 0 0 0 0 0 0 ] 1.25;
                Mw Mw [ 1 0 0 0 -1 0 0 ] 18;
	}
}
''
But it does not make sense to me since H2O is the solvent so it should not appear. Should I put **Phase 2 inertMultiComponentMixture ** and then not fill the part in solution species ?
Also another question, how are Phase 1 and Phase 2 read ? My understanding is that Phase 1 is the first species in solution species and Phase 2 is the second in solution species but i am not sure ? Can you clear me this doubt please ?

Thanks again for the time and help,

Sincerely,
Santhosh

**Reply by julienmaes** [reply]
H2O should not appear. This is fine as long as H2O does not go to the gas phase, if it does (drying) then the method we use in GCFoam does not apply

**Reply by Santhosh91000** [reply]
Hmm I see, thanks for the confirmation. This allow me to understand better,

So here, since at first I don't take into account chemical reactions, CO2dissolved will be the only solution species present in water.
Phase 1 and Phase 2 are referring to the two fluids phases (CO2 and water) and are considered as ''inertMulticomponentMixture''.
Therefore I should add a file in 0 folder for CO2dissolved for initial and boundary conditions.

In the case of chemical reactions. Since one of the two phase is water and is the place where the reactions take place I should name that phase ''Phreeqemixture'' and mention the information (D1,D2,H,Mw) for corresponding species (H+,CO32-, ...) in the ''solutionSpecies'' part but also mention initial and boundary conditions in the 0 folder.

Please let me know if this is correct.

Thanks again for the availability and assistance.

Sincerely,
Santhosh

**Reply by julienmaes** [reply]
Looks correct. Good luck!

**Comment by Santhosh91000** [comment]
Thanks a lot!
